extensions/adobe/experience/adcloud/dsp/account.schema.json

```diff
@@ -0,0 +1,92 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/dsp/account",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Advertising Cloud DSP Account",
```

Comment:
XDM Team recommends not to include company names in title. You can change the title to "DSP Advertising ..." and see what comments we get.

Reply:
Updated